### ISSUE TITLE
Cluster Configuration Support

### ISSUE THREAD

**Issue by elliottll** [HadesArchitect/GrafanaCassandraDatasource#112]
Currently the plugin only supports a single string for cluster host config.
This should allow multiple hosts in order to leverage the cluster rather than a single node.

backend/datasource.go#L284

```go
func (ds *CassandraDatasource) Connect(host, ...) (bool, error) {
        ...
	cluster := gocql.NewCluster(host)
```

![image](https://user-images.githubusercontent.com/87797182/126594199-cce21ae0-3c54-46f1-9645-de0de1e75cb8.png)

**Comment by HadesArchitect** [comment]
Hi @elliottll, thank you for your contribution. 

Most of the Cassandra drivers use the contact point only for the **initial** connection, load the cluster token ranges and then contact the proper replica nodes, so-called token-aware drivers. It's a common misunderstanding that Cassandra works only with the contact point (like some other DBMS do). 

Although I'm not 100% sure about gocql driver, so I'll check if it's token-aware.

**Comment by HadesArchitect** [comment]
Right, GoCQL is token-aware https://github.com/gocql/gocql/blob/master/policies.go#L372

But it may be a good idea to support more than one contact point indeed, I won't close this issue right now. @futuarmo Do you think we have to support multiple contact points?

**Comment by HadesArchitect** [comment]
Implemented in #130, to be merged as soon as conflicts will be resolved.
